affiliate/src/productCard/Edit.jsx

```diff
@@ -98,11 +108,18 @@ const Edit = ({ attributes, setAttributes, productSearchPageUrl }) => {
                 label="Display Description"
                 onChange={setDisplayDescription}
               />
+              <TextControl
+                help="Text for product card banner/award"
+                label="Banner Text"
```

Comment:
@Mwindo I'm not sure how we want to market this feature. Is it "Banner" or "Award"? I just put whatever here, expecting we would want to change it. (I think I would also rather use "Banner" over "Banner Text" for the label).

Reply:
Banner is more agnostic, so let's go with that. I like Banner Text slightly more since it is more descriptive, but Banner or Banner Text is fine with me!